### PR TITLE
Fix broken workflow

### PR DESCRIPTION
**Pull request by Briancbn**
- A simple fix to the broken workflow caused by https://github.com/osrf/traffic_editor/pull/149
- ament_python rosdep not found

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -9,9 +9,13 @@ jobs:
     steps:
     - name: deps
       run: |
+        sudo apt-get update
```
Comment (Yadunund) [inline .github/workflows/build.yaml]:
```suggestion
        sudo apt update
```

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -9,9 +9,13 @@ jobs:
...
+        sudo apt-get install -y wget
```
Comment (Yadunund) [inline .github/workflows/build.yaml]:
```suggestion
        sudo apt install -y wget
```

File: .github/workflows/build.yaml [hunk 3]
```diff
@@ -9,9 +9,13 @@ jobs:
...
+        echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
+        wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
         sudo apt update
         sudo apt-get install python3-shapely python3-yaml python3-requests \
```
Comment (Yadunund) [inline .github/workflows/build.yaml]:
```suggestion
        sudo apt install python3-shapely python3-yaml python3-requests \
```

File: .github/workflows/build.yaml [hunk 4]
```diff
@@ -30,7 +34,7 @@ jobs:
       run: |
         cd /home/ws_rmf
         rosdep update
-        rosdep install --from-paths src --ignore-src --rosdistro eloquent -y --skip-keys "ament_python"
+        rosdep install --from-paths src --ignore-src --rosdistro eloquent -y --skip-keys "libignition-common3 libignition-plugin-dev"
```
Comment (Yadunund) [inline .github/workflows/build.yaml]:
```suggestion
        rosdep install --from-paths src --ignore-src --rosdistro eloquent -yr
```

File: .github/workflows/build.yaml [hunk 5]
```diff
@@ -9,9 +9,13 @@ jobs:
     steps:
     - name: deps
       run: |
+        sudo apt-get update
+        sudo apt-get install -y wget
+        echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
+        wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
         sudo apt update
```
Comment (Yadunund) [inline .github/workflows/build.yaml:16]:
```suggestion
```

**Review by Yadunund — CHANGES_REQUESTED** [review]
Thanks for quickly addressing the new dependency in `traffic_editor`. While a permanent fix will be to add `libignition-common3` to the list of rosdep repo list, this is a good workaround in the meanwhile. Left some minor suggestions that can be addressed before merging.

File: .github/workflows/build.yaml (hunk 5 above)
Comment (Yadunund) [reply .github/workflows/build.yaml:16]: Sorry ignore this suggestion. We will need to update after adding the new gazebo sources list.

**Comment by Briancbn** [comment]
Thanks Yadu! Added requested changes and rebased to latest on master

**Review by Yadunund — APPROVED** [review]
Thanks @Briancbn !

**Comment by Briancbn** [comment]
Sorry, workflow still failing due to missing gcc-8. https://github.com/osrf/rmf_core/issues/113

**Comment by Yadunund** [comment]
We are in the process of updating the build instructions across the RMF repositories to match the new API changes in `rmf_core`.

**Pull request merged**